.github/workflows/update_gdocs_data.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Google Docs Data
```

Comment:
@krivard FYI - not just releases but updating the code from the docs update is automatable via this action.

Reply:
once it is deployed, I will add docu/link to the Google Doc with instructions